cypress/integration/MenuItem/position/index.js

```diff
@@ -28,7 +28,10 @@ When('the user hovers over the MenuItem', () => {
 Then(
     'the right of the MenuItem is aligned with the left of the SubMenu',
     () => {
-        getMenuItemAndSubMenuRects().then(([menuItemRect, subMenuRect]) => {
+        cy.getPositionsBySelectors(
+            '[data-test="dhis2-uicore-menuitem"]',
+            '[data-test="submenu"]'
+        ).should(([menuItemRect, subMenuRect]) => {
```

Comment:
I like the approach you've taken here. When you reworked the Select position tests, you chose a slightly different approach. Could you align the Select position tests https://github.com/dhis2/ui/pull/66/commits/f355ea6889e7217f91bdbf7cfd932be2bc0063a8 with the current approach?

Reply:
I've removed this change. Created an issue to address this here: https://github.com/dhis2/ui/issues/106